JasmineTests/Tetris/ViewModels/TetrisGameViewModelTests.swift

```diff
@@ -139,9 +144,44 @@ class TetrisGameViewModelTests: RealmTestCase {
         print(destroyedAndShiftedTiles.map { $0.destroyedTiles })
         XCTAssertEqual(Set(destroyedAndShiftedTiles.map { $0.destroyedTiles }.flatMap { $0 }), landedCoordinates,
                        "Tiles destroyed are wrong")
-        XCTAssertEqual(viewModel.currentScore, 8)
+        XCTAssertEqual(viewModel.currentScore, Constants.Game.Tetris.scoreIncrement * 2)
         XCTAssertEqual(scoreUpdateDelegateMock.scoreUpdatedCount, scoreUpdatedCount + 2,
-                       "Score did not get update in VC")
+                       "Score did not get updated in VC")
+    }
+
+    func testDestroyTilesWithTextMatchingMatchedPhrase() {
+        let viewModel = TetrisGameViewModel(gameData: gameData)
+
+        let testWords = testPhrases.flatMap { $0 }
+        for _ in 0..<testWords.count - 1 {
+            let landingCoordinate = Coordinate(row: viewModel.gridData.numRows - 1,
+                                               col: viewModel.gridData.numColumns - 1)
+            _ = viewModel.landTile(at: landingCoordinate)
```

Comment:
i dont want to do that just for tests though, good to leave it on in case the actual vm forgets to use it.